zipkin-ui/test/component_data/default.test.js

```diff
@@ -0,0 +1,32 @@
+import {
```

Comment:
We don't need multiline style here unless we import more than ~4 four items (I think).

```javascript
import {a} from 'x';
import {a, b} from 'y';
import {a, b, c} from 'z';
import {
  a,
  b,
  c,
  d
} from 'zzz';
```

WDYT?